### ISSUE TITLE
Danish Braille table for entering of Braille is not in NVDA

### ISSUE THREAD

**Issue by andersbp** [nvaccess/nvda#6773]
The Danish Braille table for entering of Braille is missing in NVDA 2016.04. I fine can select Danish Braille table for reading Braille.

**Comment by josephsl** [comment]
Hi, if there is at least one Danish computer braille table we can use, then it can be included. In case of contracted braille input, that’s covered by #2439. Thanks.

**Comment by andersbp** [comment]
Hi

 

There is more Danish Braille tables. If you can use Danish eight computer braille, I will mean that it is fine.

 

Regards,

Anders

Fra: Joseph Lee [mailto:notifications@github.com] 
Sendt: 20. januar 2017 17:39
Til: nvaccess/nvda <nvda@noreply.github.com>
Cc: andersbp <anders@boholdt-petersen.dk>; Author <author@noreply.github.com>
Emne: Re: [nvaccess/nvda] Danish Braille table for entering of Braille is not in NVDA (#6773)

 

Hi, if there is at least one Danish computer braille table we can use, then it can be included. In case of contracted braille input, that’s covered by #2439. Thanks.

**Comment by josephsl** [comment]
Hi, this is something that we should ask on the Liblouis list to find out what table can be used for braille input at this time. Thanks.

**Comment by andersbp** [comment]
Hi

 

Do you will asking on the Liblouis about this for me?

 

Thanks in advance.

 

Regards,

Anders

Fra: Joseph Lee [mailto:notifications@github.com] 
Sendt: 20. januar 2017 19:03
Til: nvaccess/nvda <nvda@noreply.github.com>
Cc: andersbp <anders@boholdt-petersen.dk>; Author <author@noreply.github.com>
Emne: Re: [nvaccess/nvda] Danish Braille table for entering of Braille is not in NVDA (#6773)

 

Hi, this is something that we should ask on the Liblouis list to find out what table can be used for braille input at this time. Thanks.

**Comment by josephsl** [comment]
Hi, ideally, you should, but for this case, I will. Thanks.

**Comment by chojiro** [comment]
Seems as if the braille tables for Danish actually work, including 6 and 8 dot contracted and uncontracted, as well as computer braille, both for input and output, though the computer braille table for some reason isn't in braille.py. Also, looks like none of the tables were used to input braille that were already in the file, but only used for output. They're not showing.

It does appear that a few of the table cause issues, like the announcement "st" to be spoken when entering characters using eight dot contracted braille.

Any information on this design implementation would be appreciated. I haven't heard of any issues so far regarding LibLouis and Danish braille.

**Comment by egli** [comment]
I'm not quite clear what you're after but as far as Danish computer braille there is `da-dk-g08.ctb` by Bue Vester-Andersen, which according to the comment is the "Danish table for 8 dots computer Braille".

**Comment by chojiro** [comment]
Sure. It's the correct table, but as in my previous comment above, I wanted to see why they weren't enabled for input and why computer Braille originally isn't included in the file at all right now. The table seems to work fine, but I have tried reaching out to Bue for a while to find out anything, since he is the maintainer. Barring the slight annoyances with some of the tables, with no forthcoming answer it should be alright for a pull request since it's a very minor change. @andersbp originally seems to want to let people know that input currently does not work for the reason mentioned. So, all there is to say is that it essentially exists, it just isn't enabled for input apparently.

**Comment by jcsteh** [comment]
Danish tables can now be selected for input in master thanks to #6449. I'll add the computer braille table in an upcoming pull request. Closing.
